.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: CI python selenium
+on:
+  pull_request:
```

Comment:
https://github.com/browserstack/BStackAutomation/blob/1fa3e8eacbba942a8b7a78910b136ab9551679cb/.github/workflows/nw_linter.yml

Can we try something like this and check if it works?

Reply:
yes, will pick